app/Kconfig.user

```diff
@@ -0,0 +1,289 @@
+menu "Global Configuration Parameters"
```

Comment:
Can I please suggest that this goes into the `components/platform/Kconfig` to match the esp32 branch? And we probably include the word "NodeMCU" in the menu name to make it easier to spot among the IDF things (especially on the esp32 IDF these days).